src/iosrv/sched.c

```diff
@@ -640,6 +648,22 @@ check_space_pressure(struct dss_xstream *dx, struct sched_pool_info *spi)
 
 		spi->spi_pressure_ts = info->si_cur_ts;
 	}
+
+//	The SCM and NVME metrics are based on the pool_id.
+//	One way to store the metrics for each pool, is to use the pool ID string
+//	in the metric name itself.  For the demo, I avoided using the pool ID
+//	because of the visual clutter from the number of pools created.
+//	d_tm_set_gauge(&scm_total, SCM_TOTAL(&vps), "IO/pool", DP_UUID(spi->spi_pool_id), "scm", "total", NULL);
+//	d_tm_set_gauge(&scm_system, SCM_SYS(&vps), "IO/pool", DP_UUID(spi->spi_pool_id), "scm", "system", NULL);
+//	d_tm_set_gauge(&scm_free, SCM_FREE(&vps), "IO/pool", DP_UUID(spi->spi_pool_id), "scm", "free", NULL);
```

Comment:
```suggestion
/* The SCM and NVME metrics are based on the pool_id. */
/* One way to store the metrics for each pool, is to use the pool ID string */
/* in the metric name itself.  For the demo, I avoided using the pool ID */
/* because of the visual clutter from the number of pools created. */
/* d_tm_set_gauge(&scm_total, SCM_TOTAL(&vps), "IO/pool", DP_UUID(spi->spi_pool_id), "scm", "total", NULL); */
/* d_tm_set_gauge(&scm_system, SCM_SYS(&vps), "IO/pool", DP_UUID(spi->spi_pool_id), "scm", "system", NULL); */
/* d_tm_set_gauge(&scm_free, SCM_FREE(&vps), "IO/pool", DP_UUID(spi->spi_pool_id), "scm", "free", NULL); */
```